src/Api/Serializer/JsonBody.php

```diff
@@ -45,22 +45,30 @@ public static function getContentType(Service $service)
      * Builds the JSON body based on an array of arguments.
      *
      * @param Shape $shape Operation being constructed
-     * @param array $args  Associative array of arguments
+     * @param array|string $args  Associative array of arguments, or a string.
      *
      * @return string
      */
-    public function build(Shape $shape, array $args)
+    public function build(Shape $shape, array|string $args)
```

Comment:
Note for the reviewer: this change accounts for document types (which are always modeled as structures).  It is possible for a document type payload value to be a string.

Reply:
Understood. I guess that, in such case, the JSON encoded payload will just be that same string.